.claude/hooks/session-start.sh

```
@@ -0,0 +1,85 @@
+if [ -f CLAUDE.md ]; then
+  vid_count=$(find . -maxdepth 2 -name 'rhythmix-*-60s' -o -name 'rhythmix-*-30s' 2>/dev/null | wc -l | awk '{print $1}')
+  rendered=$(find . -maxdepth 3 -name 'rhythmix-*.mp4' -not -path '*/node_modules/*' 2>/dev/null | wc -l | awk '{print $1}')
+  note "$vid_count rhythmix project(s), $rendered rendered MP4(s)"
+fi
+
+# --- Quick reminders ---
+if [ -d .claude/skills/rhythmix-author ]; then
+  note "rhythmix-author skill loaded — say 'make me a video' to use it"
+fi
+if [ -d .claude/commands ] && [ -n "$(ls .claude/commands/*.md 2>/dev/null)" ]; then
+  note "slash commands: $(basename -s .md $(ls .claude/commands/*.md) | tr '\n' ' ')"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)